blindsidenetworks/scalelite
# Some early attempts for RSpec test conversions.

## Description
This is an early attempt at converting the deprecated minitest test suite into RSpec. It directly converts all of the tests from one system to another, so the actual testing should be identical.

## Testing Steps
Given that these are tests, they are not tested themselves, they can be run with bundle exec rspec. To verify correctness, they should likely be run and compared to the minitest tests, as they should be a direct translation.
